__tests__/transaction.test.js

```diff
@@ -42,3 +43,46 @@ describe('transaction api', () => {
     expect(response.status).toBe(400);
   });
 });
+
+describe('transaction blocks confirmation number', () => {
+  beforeAll(async () => {
+    await TestUtils.startWallet({
+      walletId: walletIdConfirmation,
+      preCalculatedAddresses: TestUtils.addresses
+    });
+  });
+
+  afterAll(async () => {
+    await TestUtils.stopWallet({ walletId: walletIdConfirmation });
+  });
+
+  it('should return 200 with a valid body', async () => {
+    const response = await TestUtils.request
+      .get('/wallet/tx-confirmation-blocks')
+      .query({
+        id: '00000008707722cde59ac9e7f4d44efbd3a5bd5f244223816ee676d328943b1b',
+      })
+      .set({ 'x-wallet-id': walletIdConfirmation });
+    expect(response.status).toBe(200);
+    expect(response.body.success).toBe(true);
+    expect(response.body.confirmationNumber).toBe(8);
```

Comment:
Maybe add a comment that this is hard-coded from the fixtures values

Reply:
Done https://github.com/HathorNetwork/hathor-wallet-headless/pull/231/commits/bac3eb0e5f6813cdecb54e78e05116b3cc305c82